static/styles/rendered_markdown.css

```diff
@@ -40,6 +40,15 @@
         margin-top: 0;
     }
 
+    /* Changing visibility of hr in light-mode */
+
+    hr {
+        margin: 20px 0;
+        border: 0;
+        border-top: 2px solid hsl(214, 13%, 90%);
+        border-bottom: 2px solid hsl(214, 13%, 90%);
+    }
```

Comment:
With these changes, the `hr` looks significantly different (both in day-mode and night-mode) with the 2 lines becoming much more evident.
![Screenshot from 2021-07-23 00-28-42](https://user-images.githubusercontent.com/39924567/126696171-7013a505-37e9-46c6-9f93-d847156aac9a.png)

The main cause of this issue is that the `border-bottom` of `hr` in CSS has a white color, which is not visible on the white background. So, instead of increasing the border height, just changing the color of `border-bottom` of `hr` should be sufficient.

![Screenshot from 2021-07-23 00-44-26](https://user-images.githubusercontent.com/39924567/126696567-870721f7-6b7b-4aa0-ad4c-85feb377f361.png)

Without any changes
![Screenshot from 2021-07-23 00-45-22](https://user-images.githubusercontent.com/39924567/126696701-6fdf704d-0d7e-46cd-ba66-d2f11ac5a381.png)

Reply:
What thickness should I set for border-bottom then? I did this to maintain consistency as the night mode hr had the same border-top and border-bottom thickness.

Reply:
> I did this to maintain consistency as the night mode hr had the same border-top and border-bottom thickness.

I don't think so, `night-mode.css` only changes the `color` and `opacity` of `hr`, the thickness of border-bottom remains same in the night-mode.

> What thickness should I set for border-bottom then?

I think the thickness of 1px is fine. Or, you could just do these changes for `border-top` only, leaving `border-bottom` untouched (and thus, invisible in the day-mode). But then, you would need to revert these effects for the night-mode.

Reply:
I'd say, you try all these options and post the screenshots of both day-mode and night-mode on CZO and let everyone decide which looks best.

Reply:
If I apply border-bottom to day mode just changing the color and don't do the same in night mode, will that be alright?